script.py

```diff
@@ -8,16 +8,16 @@
 
 def main():
     if token:
-        response = request_vehicles(None)
-        print_info(response)
-        sys.exit(signal)
+        response = request_vehicles()
+        menu(response)
+        sys.exit(0)
```

Comment:
You don't even need to specify `0`, `sys.exit()` will set the exit code to 0 automatically.